Camera_Box_Detection_Confidence[] all_cam_data_list
